node/session.go

```diff
@@ -140,10 +140,16 @@ func (s *Session) sendFrame(frame *sentFrame) {
 
 	select {
 	case s.send <- *frame:
+		if s.node != nil {
```

Comment:
Is it a real case when `s.node` is `nil` in this context?

Reply:
I wasn't expecting s.node to be nil in this context, however [the tests were failing](https://github.com/anycable/anycable-go/runs/2220595329) without this check, so it seems there are cases where s.node could be nil in this context.

Reply:
I guess it's only a case for tests, but I would defer the final decision to @palkan

Reply:
Yeah, that reveals our testing problems 😀

I've already fixed this in my _super-secret-branch_, so, let's leave this check for now.